src/components/resource-edit/resource-edit.js

```diff
@@ -1,143 +0,0 @@
-    let needsUpdate = !isEqual(this.props.initialValues, nextProps.initialValues);
-
-    if (wasPending && needsUpdate) {
-      this.context.router.history.push(
-        `/eholdings/resources/${this.props.model.id}`,
-        { eholdings: true }
-      );
-    }
-  }
-
-  handleCancel = () => {
-    this.context.router.history.push(
-      `/eholdings/resources/${this.props.model.id}`,
-      { eholdings: true }
-    );
-  }
-
-  render() {
-    let {
-      model,
-      handleSubmit,
-      onSubmit,
-      pristine,
-      change
-    } = this.props;
-
-    let actionMenuItems = [
-      {
-        label: 'Cancel editing',
-        to: {
-          pathname: `/eholdings/resources/${model.id}`,
-          state: { eholdings: true }
-        }
-      }
-    ];
-
-    return (
-      <DetailsView
-        type="resource"
-        model={model}
-        paneTitle={model.name}
-        paneSub={model.packageName}
-        actionMenuItems={actionMenuItems}
-        bodyContent={(
-          <form onSubmit={handleSubmit(onSubmit)}>
-            <DetailsViewSection
-              label="Coverage dates"
-            >
-              <ResourceCoverageFields />
-            </DetailsViewSection>
-            <DetailsViewSection
-              label="Coverage statement"
-            >
-              <CoverageStatementFields />
-            </DetailsViewSection>
-            <DetailsViewSection
-              label="Embargo period"
-            >
-              <CustomEmbargoFields change={change} />
-            </DetailsViewSection>
-            <div className={styles['resource-edit-action-buttons']}>
-              <div
-                data-test-eholdings-resource-cancel-button
-              >
-                <Button
-                  disabled={model.update.isPending}
-                  type="button"
-                  onClick={this.handleCancel}
-                >
-                  Cancel
-                </Button>
-              </div>
-              <div
-                data-test-eholdings-resource-save-button
-              >
-                <Button
-                  disabled={pristine || model.update.isPending}
-                  type="submit"
-                  buttonStyle="primary"
-                >
-                  {model.update.isPending ? 'Saving' : 'Save'}
-                </Button>
-              </div>
-              {model.update.isPending && (
-                <Icon icon="spinner-ellipsis" />
-              )}
-            </div>
-            <NavigationModal when={!pristine && !model.update.isPending} />
-          </form>
-        )}
-      />
-    );
-  }
-}
-
-const validate = (values, props) => {
-  return Object.assign({}, validateCoverageDates(values, props), validateCoverageStatement(values), validateEmbargo(values));
-};
-
-export default reduxForm({
-  validate,
-  enableReinitialize: true,
-  form: 'ResourceEdit',
-  destroyOnUnmount: false,
-})(ResourceEdit);
```

Comment:
Where did this file and the other `resource-edit` files go? I noticed there are `custom-resource-edit` and `managed-resource-edit`, but those were pre-existing.

If I had to assume, this was left behind from another change that introduced the other edit components? Just want to make sure this wasn't an accidental delete